### DISCUSSION TITLE
Sett all option

### DISCUSSION THREAD

**Discussion by Schrotty74** [TheBobPony/MSEdgeTweaker#6] [Ideas]
Can you add a Sett All option, please?

**Comment by SmearODeer** [comment]
I Should have done that when I uploaded edits for him today. Duh Bob Duh... (Yes I'm Bob too.)

**Reply by SmearODeer** [reply]
I did add in the commit I submitted today.

**Reply by Schrotty74** [reply]
Nice thx
